README.md

```diff
@@ -38,6 +38,30 @@ This challenge is brimming with creative freedom, giving you the opportunity to
 
 ---
 
+## 👇🏼 Quick Loogies NFT Breakdown 🎨
+
+Smart contract generates dynamic SVG NFTs entirely on-chain using deterministic randomness, dynamic pricing, and real-time SVG generation.
+
+**🔄 Minting Flow:**
+
+1. User calls `mintItem()` with ETH payment that meets current price
+2. Contract generates unique traits (color, chubbiness, mouth length) and auto-increases price for next mint
+3. Payment forwarded to recipient, NFT minted to user
+
+**🎨 SVG Generation:**
+
+- `tokenURI()` creates Base64-encoded JSON metadata with embedded SVG image
+- `renderTokenById()` builds dynamic SVG art with eyes, head, and mouth positioned using trait-driven calculations
+
+**🚀 Technical Highlights:**
+
+- 🔒 Deterministic randomness: generates predictable yet unique traits for each mint
```

Comment:
Maybe add more information on where the randomness came from or a link to the contract code https://github.com/scaffold-eth/se-2-challenges/blob/svg-nft-quick-breakdown/extension/packages/hardhat/contracts/YourCollectible.sol#L46

Reply:
I think short description is enough in this case, so leaving as is for now